src/index.jsx

```diff
@@ -529,11 +529,11 @@ export default class DatePicker extends React.Component {
       this.setPreSelection(date);
     } else if (!this.props.inline) {
       if (!this.props.selectsRange) {
-        this.setOpen(false);
+        setTimeout(() => this.setOpen(false), 100);
```

Comment:
This is just a little delay on the close of the calendar popup and allows the screen reader to reads the change on the `aria-label` of the date selected